src/modules/Dropdown/Dropdown.js

```diff
@@ -134,6 +134,12 @@ export default class Dropdown extends Component {
     /** Define whether the highlighted item should be selected on blur */
     selectOnBlur: PropTypes.bool,
 
+    /** Make the dropdown options searchable (`boolean` or `function(options, searchQuery) => filteredOptions`) */
```

Comment:
Let's keep the types out of the doc blocks since they are shown formally in the docs:

![image](https://cloud.githubusercontent.com/assets/5067638/18394176/743a5a1c-766d-11e6-9f2d-bc2d878f32af.png)


Function signatures are an issue still, though, I think we'll solve these with jsdoc tags at a later time.